pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -124,8 +121,8 @@ function validateParams(vmParams) {
     // If we select installation media from URL force the user to select
     // OS, since virt-install will not detect the OS, in case we don't choose
     // to start the guest immediately.
-    if (vmParams.vendor == NOT_SPECIFIED && vmParams.sourceType == URL_SOURCE && !vmParams.startVm)
-        validationFailed['vendor'] = _("You need to select the most closely matching OS vendor and Operating System");
+    if ((vmParams.os == undefined || vmParams.os.name == 'Other OS') && vmParams.sourceType == URL_SOURCE && !vmParams.startVm)
```

Comment:
I thought the previous commit just dropped "Other OS", why is it coming back here?

Reply:
The commit that drops the 'Other OS' comes after this one,
in the first commit you still can find the 'Other OS' on the list of available OSes, and we should not allow users to choose it when the media is coming from URL.